torch/utils/data/datapipes/utils/decoder.py

```diff
@@ -228,10 +222,37 @@ def torch_audio(key, data):
             return torchaudio.load(fname)
 
 
+################################################################
+# mat
+################################################################
+class MatHandler:
```

Comment:
It seems the `ImageHandler` also has an accompanying function `imagehandler`. Maybe add a `mathandler`?

Reply:
I will change `torch_video` and `torch_audio` to `videohandler` and `audiohandler` as well.